README.md

```diff
@@ -120,3 +120,34 @@ struct Foo
     let bar: String
 }
 ```
+
+### Forced downcasts and unwrapping
+
+Avoid using `as!` to force a downcast, or `!` to force unwrapping. Use `as?` to attempt the cast, then deal with the failure case explicitly.
+
+``` swift
+// Correct
+func process(someObject: Any) {
+    guard let element = someObject as? Element else {
+        // Optional error handling goes here
+        return
+    }
+    process(element)
+}
+
+// Wrong
+func process(someObject: Any) {
+    process(someObject as! Element)
+}
+```
+
+Or use `throw` when the failure is unrecoverable
```

Comment:
I'd delete everything including and below this line, since the example below:

1. Is a sub-case of the more generic "Correct" example provided above, and
2. It specifically recommends to use exceptions (over assertions for example) without enough context to know which one is warranted.
3. An error-handling guide contribution [is coming up soon](https://github.com/wordpress-mobile/swift-style-guide/issues/8).